packages/common/src/services/utilities.ts

```diff
@@ -883,7 +883,7 @@ export function parseUtcDate(inputDateString: any, useUtc?: boolean): string {
  * @returns
  */
 export function removeAccentFromText(text: string, shouldLowerCase = false) {
-  const normalizedText = text.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
+  const normalizedText = text.normalize?text.normalize('NFD').replace(/[\u0300-\u036f]/g, ''):text;
```

Comment:
I think it's incorrect syntax isn't it? I think it should always include a dot `?.` instead of `?`

Reply:
redid the code, see if it's better this way.
![image](https://user-images.githubusercontent.com/44271732/135852796-55b309fe-247b-4fac-8ff6-6be92964e68b.png)